keywording/text.xml

```diff
@@ -421,14 +421,53 @@ dependency for your package).
 </p>
 
 <p>
-Some architectures (<c>alpha</c>, <c>ia64</c>, <c>mips</c>, <c>riscv</c>,
+Some architectures (<c>alpha</c>, <c>loong</c>, <c>mips</c>, <c>riscv</c>,
 <c>s390</c>) do not maintain a stable keyword, so packages are not to be marked
 stable for one of these architectures.
 </p>
 
 </body>
 </subsection>
 
+<subsection>
+<title>Stabilization groups</title>
+<body>
+
+<p>
+Stabilization groups are sets of packages defined to help organize and automate
+stabilization requests. These groups are stored under
+<c>metadata/stabilization-groups/</c>, and all files within this directory
+(including subdirectories) are included recursively.
+</p>
+
+<p>
+Each group is named according to its path, for example <c>@gnome/vte</c>. This
+naming convention helps avoid collisions and makes referencing groups
+straightforward. The files use a simple format: one <c>cat/pkg</c> per line,
+with lines starting with <c>#</c> treated as comments. Whitespace before and
+after entries is ignored.
+</p>
+
+<p>
+When invoking tools like <c>pkgdev bugs</c>, passing a set (anything starting
```

Comment:
I assume that only works locally, meaning I couldn't use a set directly in the stabilization bug's package list?